builtin/settingtypes.txt

```diff
@@ -452,6 +452,9 @@ fall_bobbing_amount (Fall bobbing) float 0.0
 #    -    pageflip: quadbuffer based 3d.
 3d_mode (3D mode) enum none none,anaglyph,interlaced,topbottom,sidebyside,pageflip
 
+#    Distance where the eyes converge in 3D mode.
+3d_convergence_distance (3D convergence distance) float 1.95
```

Comment:
Not quite. If we take the player model height (1.76 nodes) and equate this to world average human height a node ends up being a little less than 1 metre.
So the unit is 'nodes', we don't officially refer to metres in documentation because this is an abstract world.